### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## [Unreleased]
+
+## [0.1.0] - 2019-06-04
+
+### Added
+
+- Add push-to-app-catalog job.
+
+[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/olivierlacan/keep-a-changelog/releases/tag/v0.1.0
```
Comment (TheoBrigitte) [inline CHANGELOG.md]: Wait, why do we have this url there ?

Comment (kopiczko) [reply CHANGELOG.md]: I pushed the tag manually for the commit with job promoting to prod. I'll describe the development process here in the later PR.

Comment (kopiczko) [reply CHANGELOG.md]: Ah. And this should be `giantswarm/architect-orb`... Changing.

Comment (kopiczko) [reply CHANGELOG.md]: Good catch, thanks!

**Pull request merged**